### PR TITLE
Add static website script

### PR DESCRIPTION
**Pull request by A-Harby**
### Description

Add the static website example script.

### Changes

static website ts.

### Related Issues

https://github.com/threefoldtech/tfgrid-sdk-ts/issues/2785

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/grid_client/scripts/applications/static_website.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import { FilterOptions, GatewayNameModel, MachinesModel } from "../../src";
+import { config, getClient } from "../client_loader";
+import { log, pingNodes } from "../utils";
+
+async function deploy(client, vms, subdomain, gatewayNode) {
```
Comment (Mahmoud-Emad) [inline packages/grid_client/scripts/applications/static_website.ts]: Let's define the type of the parameters please, and let's apply it to the whole script

File: packages/grid_client/scripts/applications/static_website.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    name: subdomain,
+    node_id: gatewayNode.nodeId,
+    tls_passthrough: false,
+    backends: ["http://[" + vmPlanetary + "]:9000"],
+  };
+
+  const resultGateway = await client.gateway.deploy_name(gw);
+  log("================= Deploying name gateway =================");
+  log(resultGateway);
+  log("================= Deploying name gateway =================");
```
Comment (Mahmoud-Emad) [inline packages/grid_client/scripts/applications/static_website.ts]: it'll be good if we moved this part to another function

**Review by Mahmoud-Emad — CHANGES_REQUESTED** [review]
Good job ya 7arbii

File: packages/grid_client/scripts/applications/static_website.ts (hunk 2 above)
Comment (A-Harby) [reply packages/grid_client/scripts/applications/static_website.ts]: To follow the other scripts at https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2743#discussion_r1607723148 and keep things consistent, we should leave it as it's.

I also added some comments for better reliability.

**Review by Mahmoud-Emad — APPROVED** [review]

File: packages/grid_client/scripts/applications/static_website.ts (hunk 2 above)
Comment (Mahmoud-Emad) [reply packages/grid_client/scripts/applications/static_website.ts]: indeed xD

**Review by khaledyoussef24 — APPROVED** [review]
good job harby

**Pull request merged**